SConstruct

```diff
@@ -1001,9 +1001,16 @@ if env['googletest'] == 'none':
 
 # Check for Eigen and checkout submodule if needed
 if env['system_eigen'] in ('y', 'default'):
-    if conf.CheckCXXHeader('Eigen/Dense', '<>'):
+    if conf.CheckCXXHeader('eigen3/Eigen/Dense', '<>'):
```

Comment:
As @speth mentioned, please use double quotes `"` in all Python code. SCons scripts count too 😊